### PR TITLE
Ver 1.0.1 done

### PR DESCRIPTION
**Pull request by boefraty74**

### CODE REVIEW

File: pbiviz.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "visual": {
     "name": "PowerBI-visuals-forcasting-TBATS",
     "displayName": "Forecasting TBATS",
-    "guid": "PBI_CV_8EDDC07B_TBATS_WORK",
+    "guid": "PBI_CV_8EDDC07B_TBATS",
```
Comment (AviSander) [inline pbiviz.json]: why this change?

File: src/visual.ts [hunk 2]
```diff
@@ -252,8 +252,8 @@ module powerbi.extensibility.visual {
                         objectName: objectName,
                         properties: {
                             valuesNonNegative: this.settings.settings_additional_params.valuesNonNegative,
-                            //  algModeFast: this.settings_additional_params.algModeFast,
-                            //  useParProc: this.settings_additional_params.useParProc
+                            //  algModeFast: this.settings_additional_params.algModeFast,//full mode
```
Comment (AviSander) [inline src/visual.ts]: // full mode... ?

**Review by AviSander — CHANGES_REQUESTED** [review]
see comments

File: pbiviz.json (hunk 1 above)
Comment (boefraty74) [reply pbiviz.json]: _WORK is debug guid
By mistake, I pushed into master  non final version of  visual

File: src/visual.ts (hunk 2 above)
Comment (boefraty74) [reply src/visual.ts]: these lines are commented for master (AppSource)

Comment (AviSander) [reply src/visual.ts]: not sure i follow?

**Comment by boefraty74** [comment]
All done, or answered.

**Review by AviSander — APPROVED** [review]
